dashboard/app/models/level_group.rb

```diff
@@ -87,4 +88,135 @@ def pages
   def plc_evaluation?
     levels.map(&:class).uniq == [EvaluationMulti]
   end
+
+  # Surveys: How many students must complete a survey before any results are shown.
+  SURVEY_REQUIRED_SUBMISSION_COUNT = 5
+
+  # Surveys: Given a sublevel, and the known response to it, return a result object.
```

Comment:
Ah, I thought other levels could have sublevels?  E.g. multi embedded in app lab.